client/common/cmdline.c

```diff
@@ -1474,6 +1474,7 @@ int freerdp_client_settings_parse_command_line_arguments(rdpSettings* settings,
 
 				CommandLineSwitchCase(arg, "v")
 		{
+			free (settings->ServerHostname);
```

Comment:
settings->ServerHostname should be NULLed here or we might end up with a double free in case strchr below fails